module3/lessons/react_ideabox.md

```diff
@@ -936,7 +955,7 @@ function App(){
   return(
```

Comment:
Not sure if the intent is to leave the `console.log(id);` in the code, but something I wanted to call out just in case since it's never talked about and there aren't any logs anywhere else.